Swerve/SwerveTemplates/VisionBaseSwerve.java

```diff
@@ -25,9 +30,10 @@ public abstract class VisionBaseSwerve extends BaseSwerve {
     protected final SwerveDrivePoseEstimator odom;
     private final SwerveDriveKinematics kinematics;
     private ArrayList<Camera> cameras;
+    private Limelight limelight;
 
-    public VisionBaseSwerve(SwerveModule[] swerveMods, Gyro gyro, SwerveDriveKinematics kinematics,
-            Matrix<N3, N1> stateStdDevs, Matrix<N3, N1> visionMeasurementStdDevs, List<CameraConfig> cameraConfigs) {
+    public VisionBaseSwerve(SwerveModule[] swerveMods, Gyro gyro, SwerveDriveKinematics kinematics, Matrix<N3, N1> stateStdDevs, 
+                            Matrix<N3, N1> visionMeasurementStdDevs, List<CameraConfig> cameraConfigs, LimelightConfig limelightConfig) {
```

Comment:
make this a list? can give more flexibility in future use cases

Reply:
done